deepinterpolation/generator_collection.py

```diff
@@ -536,126 +398,163 @@ def __data_generation__(self, local_x, local_y, local_z, local_t):
         return input_full, output_full
 
 
-class EphysGenerator(DeepGenerator):
-    "Generates data for Keras"
+class SequentialGenerator(DeepGenerator):
+    """This generator stores shared code across generators that have a
```

Comment:
The docstring says that this class is abstract, but in order for it to be abstract, it needs to subclass `abc.ABC` and declare atleast 1 method decorated with `@abc.abstractmethod`.

Reply:
True, I edited. It can't be an abstract class as I have to inherit to from a Sequence class.